src/renderer/extensions/vueNodes/widgets/composables/useStepperControl.ts

```diff
@@ -0,0 +1,117 @@
+import { computed, onMounted, onUnmounted, ref } from 'vue'
+import type { Ref } from 'vue'
+
+import { useGlobalSeedStore } from '@/stores/globalSeedStore'
+import type { ControlOptions } from '@/types/simplifiedWidget'
+
+import { numberControlRegistry } from '../services/NumberControlRegistry'
+
+export enum NumberControlMode {
+  FIXED = 'fixed',
+  INCREMENT = 'increment',
+  DECREMENT = 'decrement',
+  RANDOMIZE = 'randomize',
+  LINK_TO_GLOBAL = 'linkToGlobal'
+}
```

Comment:
```suggestion
export const NumberControlMode = {
  FIXED: 'fixed',
  INCREMENT: 'increment',
  DECREMENT: 'decrement',
  RANDOMIZE: 'randomize',
  LINK_TO_GLOBAL: 'linkToGlobal'
} as const
```

That thing we talked about